### PR TITLE
task/11_create_basic_structure_and_classes

### PR DESCRIPTION
**Pull request by brunobialas**

### CODE REVIEW

File: src/main/java/Model/Address.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    String street;
+    String houseNumber;
+    String postalCode;
+
+    public Address(String street, String houseNumber, String postalCode) {
+        this.street = street;
+        this.houseNumber = houseNumber;
+        this.postalCode = postalCode;
+    }
+
```
Comment (jakubbialass) [inline src/main/java/Model/Address.java]: Delete that line

File: src/main/java/Model/Product.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+public Product(String name, String description, Float value) {
+    this.name = name;
+    this.description = description;
+    this.value = value;
+    }
+
+
+
+
```
Comment (jakubbialass) [inline src/main/java/Model/Product.java]: Remove those lines

File: src/main/java/Model/User.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        this.name = name;
+        this.surname = surname;
+        this.address = adress;
+        this.points = points;
+        this.email = email;
+        this.password = password;
+    }
+
+
+
```
Comment (jakubbialass) [inline src/main/java/Model/User.java]: Remove those lines

File: src/main/java/Model/Product.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package Model;
+
+
```
Comment (jakubbialass) [inline src/main/java/Model/Product.java]: Remove those lines

File: src/main/java/Model/Basket.java [hunk 5]
```diff
@@ -0,0 +1,8 @@
+package Model;
+
+import java.util.ArrayList;
+
+public class Basket {
+    Float summaryValue;
+    ArrayList<Product> basket = new ArrayList<>();
```
Comment (jakubbialass) [inline src/main/java/Model/Basket.java]: Don't initialize variable here. It should be done eg. in the constructor.

**Review by jakubbialass — CHANGES_REQUESTED** [review]

File: src/main/java/Model/Address.java (hunk 1 above)
Comment (brunobialas) [reply src/main/java/Model/Address.java]: Done.

File: src/main/java/Model/Product.java (hunk 2 above)
Comment (brunobialas) [reply src/main/java/Model/Product.java]: Done.

File: src/main/java/Model/User.java (hunk 3 above)
Comment (brunobialas) [reply src/main/java/Model/User.java]: Done.

File: src/main/java/Model/Product.java (hunk 4 above)
Comment (brunobialas) [reply src/main/java/Model/Product.java]: Done.

File: src/main/java/Model/Basket.java (hunk 5 above)
Comment (brunobialas) [reply src/main/java/Model/Basket.java]: Done.

File: src/main/java/Model/Product.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+package Model;
+public class Product {
+    private String name;
+    private String description;
+    private Float value;
+    private Boolean isOnSale = false;
```
Comment (jakubbialass) [inline src/main/java/Model/Product.java]: Initialize this boolean in the constructor

**Review by jakubbialass — CHANGES_REQUESTED** [review]

File: src/main/java/Model/Product.java (hunk 6 above)
Comment (brunobialas) [reply src/main/java/Model/Product.java]: Done

**Review by jakubbialass — APPROVED** [review]

**Pull request merged**